image_search_engine/src/model/clip.py

```diff
@@ -0,0 +1,90 @@
+import torch
+from torch import nn
+import torch.nn.functional as F
+import timm
+from image_search_engine.src import config as CFG
+from transformers import DistilBertConfig, DistilBertModel
+
+class Clip(nn.Module):
+    def __init__(
+        self,
+        temperature=CFG.TEMPERATURE,
```

Comment:
Like the previous comment, all of these should be parameters and consts should be passed as arguments.